src/web/configurators/express.js

```diff
@@ -9,6 +9,7 @@ var stringUtils = require('../../utils'),
 module.exports = function(container, libs, next) {
 	var app = container.resolveSync('App'),
 		config = container.resolveSync('Config'),
+		options = container.resolveSync('ExpressOptions') || {},
```

Comment:
I don't really know what resolveSync does if the value isn't found, so I figured this might work. Any good ways to test this?